examples/games/mafia_werewolf/play_mafia.qnt

```diff
@@ -0,0 +1,185 @@
+// -*- mode: Bluespec; -*-
+
+module mafia {
```

Comment:
Can you add a nice header to this file linking to some reference that explains the game and adding your name as the author? You can take inspiration from some other example like tictactoe.

I forgot to mention this in the Rock Paper Scissor one, but it would also be nice to have some header for that. We have a readme, but it would be useful to have a short description and your name as the author.